go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.27.43
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.65.3
 	github.com/aws/smithy-go v1.22.0
-	github.com/eggsampler/acme/v3 v3.6.2-0.20250208073118-0466a0230941
+	github.com/eggsampler/acme/v3 v3.6.2
```

Comment:
While I appreciate this bump to an actually-released version, it doesn't seem to actually affect any of our vendored code. That suggests to me that it isn't actually necessary for the rest of this PR to work, and I think it should be separated out.